plugin/db/driver.go

```diff
@@ -244,8 +244,8 @@ type MigrationInfo struct {
 }
 
 // placeholderRegexp is the regexp for placeholder.
-// Refer to https://stackoverflow.com/a/6222235/19075342, but we support '.' for now.
-const placeholderRegexp = `[^\\/?%*:|"<>]+`
+// Refer to https://stackoverflow.com/a/6222235/19075342, but we support "." and "*" for now.
+const placeholderRegexp = `[^\\/?%:|"<>]+`
```

Comment:
We require `{{DB_NAME}}` to be presented in the file path template so allowing the use of `*` as the database name to match all databases makes logical sense, so a DDL file path would look like `*__0001__migrate__add_table.sql`

Alternatively, we can remove the requirement of the `{{DB_NAME}}` placeholder in the file path template.